### ISSUE TITLE
loss function?

### ISSUE THREAD

**Issue by ZeratuuLL** [sksq96/pytorch-vae#2]
I am not sure if I am correct. It seems that the loss function is directly a copy from the pytorch's official implement of VAE. But it seems to be the opposite number of the loss in the function.

In the paper the loss should be -DKL+log p(x|z)
and here are we minimizing DKL - log p(x|z)?

I am not sure if I understand correctly about ```F.binary_cross_entropy``` but from the document of ```nn.BCELoss``` [torch.nn.functional.binary_cross_entropy](https://pytorch.org/docs/stable/nn.html#binary-cross-entropy) it should be returning -log p(x|z).....

**Comment by ZeratuuLL** [comment]
I am stupid. We should maximize the loss....
